### PR TITLE
chore: Bump node-simctl to version 6

### PR DESCRIPTION
**Pull request by mykola-mokhnach**

### CODE REVIEW

File: test/helpers/device-helpers.js [hunk 1]
```diff
@@ -1,24 +1,34 @@
...
-async function createDevice (opts = {}) {
+async function prepareDevice (opts = {}) {
   const {
     model = MODEL,
     platformVersion = PLATFORM_VERSION,
+    prebooted = true,
   } = opts;
-  return await simctl.createDevice(`appium-idb-tests-${UUID.create().hex.toUpperCase()}`,
+  const simctl = new Simctl();
+  simctl.udid = await simctl.createDevice(
```
Comment (imurchie) [inline test/helpers/device-helpers.js]: Why is this needed, and not done in the `createDevice` function?

Comment (mykola-mokhnach) [reply test/helpers/device-helpers.js]: I wanted to make it more explicit for the case createDevice builds multiple devices in a loop

Comment (imurchie) [reply test/helpers/device-helpers.js]: Ah. ok.
